addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -484,7 +484,7 @@
         * @private
         */
        _afterJoin: function () {
-            this.isMember = true;
+            this.trigger_up('join_course'); // Update all slides from parent
```

Comment:
Would put that comment in the parent's docstring as this is not this widget's responsibility.

Also, as I said on the original PR, it's possible to answer / join from the non-fullscreen view:

![image](https://user-images.githubusercontent.com/41282536/198252081-7a818237-b18e-44af-98e8-d9b5f6bb1b13.png)

(You can just click on the "Exit fullscreen" button to get there).

It would be interesting to check if the issue does not also affect the "non-fullscreen mode".
Probably not as the "Next" button redirects to a brand new page.

Reply:
In the case of non fullscreen mode, the frontend is correctly updated.

On the other hand, we must be registered before being able to answer the questions in non fullscreen mode (unless I forgot something). We arrive on the page in non-fullscreen mode, we must click on the "join & submit" button, the display of the button is updated (which we must check for this fix), but we must refresh the page to be able to answer the questions.